addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -511,8 +511,9 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
             return;
         }
         // Add Action select
-        const firstOption = uiFragment.querySelector(':first-child');
-        uiFragment.insertBefore(this.selectActionEl.cloneNode(true), firstOption);
+        const firstOption = uiFragment.childNodes[0];
+        const parentOfFirstOption = firstOption.parentNode;
+        parentOfFirstOption.insertBefore(this.selectActionEl.cloneNode(true), firstOption);
```

Comment:
Ok for the `childNodes[0]`. But is the `parentOfFirstOption` useful? Surely even in safari `uiFragment.childNodes[0].parentNode` is equal to `uiFragment`?

Reply:
Wow, well done! The real problem was that the insertBefore function needs to be called from the parent of the referenceNode. I though I fixed it by using parentNode of fisrtOption but then we identified an issue on where the options were inserted. This is why I introduced the ChildNodes[0].

So you are totally right, the first change I made became useless! Sorry for that!